x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/10_keyword.yml

```diff
@@ -155,6 +155,21 @@ setup:
   - match: {hits.total.value: 1}
   - match: {hits.hits.0._source.voltage: 5.8}
 
+---
+"explain term query":
+  - do:
+      search:
+        index: sensor
+        explain: true
+        body:
+          query:
+            term:
+              day_of_week: Monday
+  - match: {hits.hits.0._explanation.description: day_of_week:Monday}
+  - match: {hits.hits.0._explanation.details.0.value: 1.0}
+  - match: {hits.hits.0._explanation.details.0.description: 'runtime field query scoring boost'}
+  - match: {hits.hits.0._explanation.details.0.details: []}
+
```

Comment:
I went with fairly simple integration tests for this instead of unit tests because it felt easier. I could go either way, but I think this gets the job done.

Reply:
thats fine with me, maybe a small unit test for the lucene query would be also useful?